test/list-members.test.js

```diff
@@ -162,6 +162,8 @@ test('listMembers works with exclusion', async (t) => {
       '00000000000000000000000000000000000000000000000000000000000ca201',
       'hex'
     ),
+    timeoutLow: 0.5,
+    timeoutHigh: 0.7,
```

Comment:
just wrote this in chat about this test:

hmm quick hunch in one test:

alice adds bob and carol
alice removes bob
carol creates a new epoch too for some reason
should be fine in theory but alice doesn't get that epoch, and when alice later adds david, it's only to her own epoch
then when alice gets carol's epoch, there's a conflict
alice picks carol's epoch as preferred since it has fewer members

-> wrong member list, missing david

easiest "fix" for all this is just to disable recovery in most tests (set timeout high (ie default)). then selectively enable in a few cases that we actually want to test

i might just also switch from this to doing the demo app. would be fun to show in berlin. and would show us what bugs actually exist IRL